symphony/lib/core/class.symphony.php

```diff
@@ -228,7 +228,8 @@ public static function initialiseLog($filename = null)
 
         self::$Log = new Log($filename);
         self::$Log->setArchive((self::Configuration()->get('archive', 'log') == '1' ? true : false));
-        self::$Log->setMaxSize(intval(self::Configuration()->get('maxsize', 'log')));
```

Comment:
Removing this `intval` check assumes that log files will always be written by Symphony, I know I've definitely edited these files through nano or so in the past, so it might be good to keep the training wheels here for safety (given it's LTS).

Reply:
This has been moved inside the `setMaxSize` function.

Reply:
Woops, missed that. Disregard! Sorry!